EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -241,29 +241,29 @@ QUEST_LV_0300_20151001_000240	Even if you escape, you're still a traitor.{nl}You
 QUEST_LV_0300_20151001_000241	If you cooperate towards my death, I will definitely remove your curse.{nl}I promise in the name of Giltine.
 QUEST_LV_0300_20151001_000242	It doesn't matter what happens to my body.{nl}I just want to rest...
 QUEST_LV_0300_20151001_000243	The poisonous plant that grows up there would be able to take my life away.
-QUEST_LV_0300_20151001_000244	I will pray that a plant that is poisonous to humans brings me eternal sleep.
+QUEST_LV_0300_20151001_000244	I will pray that a plant that is poisonous to humans brings me eternal rest.
 QUEST_LV_0300_20151001_000245	Thanks. Now, I will pray for the end of my agonizing life.
 QUEST_LV_0300_20151001_000246	So nothing occurs as I expected.{nl}If that's the case... We can't say we are in distress. We just have to rely on the one that we demons want to eliminate.
-QUEST_LV_0300_20151001_000247	Please bring the divine water from the one who follows the call of the goddesses.{nl}That sacred power will lead me to death.
-QUEST_LV_0300_20151001_000248	I heard the Cleric Master handles that divine water.{nl}I don't know if he's going to give me any.
-QUEST_LV_0300_20151001_000249	Are you talking about divine water?{nl}I am sorry, but may I ask you where you are going to use it.
-QUEST_LV_0300_20151001_000250	There are demons who need the divine water?{nl}I am sorry, but I can't help you.{nl}
+QUEST_LV_0300_20151001_000247	Please bring the holy water from the one who follows the call of the goddesses.{nl}That sacred power will lead me to death.
+QUEST_LV_0300_20151001_000248	I heard the Cleric Master handles the holy water.{nl}I don't know if he's going to give me any.
+QUEST_LV_0300_20151001_000249	Are you talking about holy water?{nl}I am sorry, but may I ask you where you are going to use it.
+QUEST_LV_0300_20151001_000250	There are demons who need the holy water?{nl}I am sorry, but I can't help you.{nl}
 QUEST_LV_0300_20151001_000251	I am not going to be involved in your personal matter.{nl}No matter what the reasons are, I can't help demons' matters as a priest.
 QUEST_LV_0300_20151001_000252	As a priest, I can't cooperate on the tasks of the demons.{nl}I hope you understand this as well.
 QUEST_LV_0300_20151001_000253	So... that's it.{nl}Maybe, we came to expect something that we shouldn't expect.
 QUEST_LV_0300_20151001_000254	The people who I believed to be my colleagues betrayed me. {nl}Then, there is no reason for me to hold on to this.{nl}
 QUEST_LV_0300_20151001_000255	The demon lord has ordered me to collect the spirits of humans that will be dedicated to the god.{nl}That's why we began working on creating the integrator which would collect the spirits.{nl}
-QUEST_LV_0300_20151001_000256	The process was smooth, but I suffered from the Petrifying Frost.{nl}They laughed and told me,{nl}
+QUEST_LV_0300_20151001_000256	The process was smooth, but I suffered from the curse of the Petrifying Frost.{nl}Frozen freak, they said while laughing.{nl}
```

Comment:
The previous line is: `-QUEST_LV_0300_20151001_000256	The process was smooth, but I suffered from the Petrifying Frost.{nl}They laughed and told me,{nl}`

But it is currently: `QUEST_LV_0300_20151001_000256	The process was smooth, but I suffered from **the curse of** the Petrifying Frost.{nl}They laughed and told me,{nl}`

The reason why it is conflicting is because you're not using the latest sentence.